.github/workflows/update.yml

```
@@ -35,6 +35,10 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: '3.9'
+      - name: Install Transifex CLI
+        run: |
+          curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+          mv tx /usr/local/bin/tx
```

As far as I'm concerned this will leave a LICENSE file coming from new tx client in the working directory.

Consider installing in `/user/local/bin` directly.
```suggestion
        run: |
          curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
        working-directory: /usr/local/bin
```